src/libraries/Microsoft.Extensions.Configuration.Binder/gen/ConfigurationBindingSourceGenerator.Emitter.cs

```diff
@@ -58,27 +126,54 @@ public Emitter(SourceProductionContext context, SourceGenerationSpec generationS
 
             public void Emit()
             {
+                if (!_generationSpec.ShouldEmitMethods(Helpers.GetMethods | Helpers.BindMethods | MethodSpecifier.Configure))
```

Comment:
>Configure [](http://example.com/codeflow?start=114&length=9)

we don't need GetValue here?

Reply:
Implementation for `GetValue`'s coming in a follow up. For now, we throw `NotSupportedException`.